ui/tests/integration/components/replication-page-test.js

```diff
@@ -16,7 +21,13 @@ module('Integration | Enterprise | Component | replication-page', function(hooks
 
   test('it renders', async function(assert) {
     await render(hbs`<ReplicationPage @model={{model}} />`);
-
     assert.dom('[data-test-replication-page]').exists();
+    assert.dom('[data-test-layout-loading]').doesNotExist();
+  });
+
+  test('it renders loader when clusterId is unknown', async function(assert) {
```

Comment:
maybe you could add a test here to test when the cluster is bootstrapping as well?